sdk/test/ProtocolTests/AWSSDK.ProtocolTests.Net45.sln

```diff
@@ -0,0 +1,103 @@
+﻿
```

Comment:
Did you mean to include solution files for other target frameworks? (I'm assuming these are for running the protocol tests in our build system).

Reply:
We haven't created the multi targeted netstandard solution file yet because some of the utility projects are targeted specifically for net framework. It requires upgrades there which are in progress.